spec/features/admin/admin_deletes_themselves.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "admin deletes themselves" do
+  scenario "and is logged out" do
+    user = create(:user, admin: true)
```

Comment:
Change `user` to `admin`?

Reply:
Since, you are creating an admin a few times, look into adding an `:admin` trait to the `:user` factory.
Once the trait is in place, you can create an admin by calling: `admin = create(:user, :admin)`

Alternatively, you can define an admin factory, which is nested inside the user factory, and which sets the `admin` attribute to true. And then just call `admin = create(:admin)` in your specs.

Reply:
Very cool, Done in sha: 281932b